lutris/sysoptions.py

```diff
@@ -344,7 +348,7 @@ def get_vk_icd_choices():
     {
         "option": "vk_icd",
         "type": "choice",
-        "default": get_vk_icd_choices()[0][1],
+        "default": "",
```

Comment:
No default? That seems unfortunate.

Reply:
Defaulting to "" makes the default Auto which leaves the env var unset.

IMO this makes more sense than Lutris defaulting to setting this env var, which is the current behavior.

Reply:
I could revert this line change - get_vk_icd_choices()[0][1] -> ("Auto: the GPU name", "")[1] -> "" so it makes no difference other than being slightly faster.

Or maybe add a comment noting that this corresponds to Auto / not setting the env var to allow default device selection to happen.

Reply:
Oh, I see. Perhaps a comment then, I missed that.